package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "mozilla-idea-town-app",
+  "description": "dom twiddling for the town of ideas",
+  "version": "1.0.0",
+  "author": "Mozilla (https://mozilla.org/)",
+  "bugs": {
+    "url": "https://github.com/mozilla/idea-town-app/issues"
+  },
+  "dependencies": {
+    "ampersand": "^3.0.5",
+    "ampersand-app": "^1.0.4",
+    "ampersand-collection": "^1.4.5",
+    "ampersand-collection-view": "^1.4.0",
+    "ampersand-dom": "^1.4.0",
+    "ampersand-events": "^1.1.1",
+    "ampersand-model": "^5.0.3",
+    "ampersand-router": "^3.0.2",
+    "ampersand-state": "^4.5.6",
+    "ampersand-view": "^8.0.1",
+    "ampersand-view-switcher": "^2.0.0",
+    "babel-core": "^5.8.22",
+    "bluebird": "^2.9.34",
+    "browserify": "^11.0.1",
+    "domready": "^1.0.8",
+    "globby": "^2.1.0",
+    "gulp-if": "^1.2.5",
+    "mozilla-tabzilla": "^0.4.0",
+    "mustache": "^2.1.3",
+    "node-bourbon": "^4.2.3",
+    "node-neat": "^1.7.2",
+    "node-sass": "^3.2.0",
+    "normalize.css": "^3.0.3",
+    "through2": "^2.0.0"
+  },
+  "devDependencies": {
+    "babel": "^5.8.21",
+    "babel-eslint": "4.0.10",
+    "babelify": "^6.1.3",
+    "browserify": "^11.0.1",
```

Comment:
Honestly I don't really "get" the difference between dependencies and devDependencies in some cases/repos. Like, would I be able to clone this repo and do an `npm install --production` and NOT install devDependencies and then have anything work? Or would I need Gulp and a bunch of devDependencies in order to build assets and get started? So do we move all the devDependencies into dependencies, and only use devDeps for things like mocha/selenium/intern and other purely optional modules.